ktor-network/common/src/io/ktor/network/sockets/DatagramSendChannelBase.kt

```
@@ -0,0 +1,124 @@
+        while (true) {
+            val handler = onCloseHandler.value
+            if (handler === CLOSED_INVOKED) break
+            if (handler == null) {
+                if (onCloseHandler.compareAndSet(null, CLOSED)) break
+                continue
+            }
+
+            require(onCloseHandler.compareAndSet(handler, CLOSED_INVOKED))
+            handler(closedCause.value)
+            break
+        }
```

detekt (1.23.8)

[warning] This loop contains an unconditional jump expression which essentially renders it useless as it will exit the loop during the first iteration.

(detekt.potential-bugs.UnconditionalJumpStatementInLoop)